src/ReactWrapper.js

```diff
@@ -468,18 +468,18 @@ export default class ReactWrapper {
    * testing events should be met with some skepticism.
    *
    * @param {String} event
-   * @param {Array} args
+   * @param {Object} mock (optional)
    * @returns {ReactWrapper}
    */
-  simulate(event, ...args) {
+  simulate(event, mock = {}) {
     this.single(n => {
       const mappedEvent = mapNativeEventNames(event);
       const eventFn = Simulate[mappedEvent];
       if (!eventFn) {
         throw new TypeError(`ReactWrapper::simulate() event '${event}' does not exist`);
       }
 
-      eventFn(findDOMNode(n), ...args);
+      eventFn(findDOMNode(n), mock);
```

Comment:
I might be overthinking this. But is there a chance this broke events that were dependent on second and third React arguments?

React usually sends a couple arguments to a function:
```handler(syntheticEvent, ...args, reactId, nativeEvent)```

So if anyone was depending on the `reactId` argument or the last `nativeEvent` argument we just broke their tests. I think a simple fix would be adding the `...args` back in after the mock so people can specify those events if they need to.